e2e/scenario_test.go

```diff
@@ -1048,10 +1048,16 @@ func runScenarioUbuntu2204GPU(t *testing.T, vmSize string) {
 				vmss.SKU.Name = to.Ptr(vmSize)
 			},
 			Validator: func(ctx context.Context, s *Scenario) {
-				// Ensure nvidia-modprobe install does not restart kubelet and temporarily cause node to be unschedulable
+				// First, ensure nvidia-modprobe install does not restart kubelet and temporarily cause node to be unschedulable
 				ValidateNvidiaModProbeInstalled(ctx, s)
 				ValidateKubeletHasNotStopped(ctx, s)
 				ValidateServicesDoNotRestartKubelet(ctx, s)
+
+				// Then validate NPD configuration and GPU monitoring
+				ValidateNPDGPUCountPlugin(ctx, s)
```

Comment:
we should be able to call these validators once in common linux validation based on scenario tags instead of specifying them in multiple places (can probably use the GPU tag?)